docs/tutorial/part-seven/index.md

```diff
@@ -4,13 +4,12 @@ typora-copy-images-to: ./
 disableTableOfContents: true
 ---
 
-> This tutorial is part of a series about Gatsby’s data layer. Make sure you’ve gone through [part 4](/tutorial/part-four/), [part 5](/tutorial/part-five/), and [part 6](/tutorial/part-six/) before continuing here.
+> यह ट्यूटोरियल Gatsby के डेटा लेयर के बारे में एक श्रंखला का हिस्सा है। आगे बढ़ने से पहले सुनिश्चित करें कि आप इससे पहले [भाग ४](/tutorial/part-four/), [भाग ५](/tutorial/part-five/), और [भाग ६](/tutorial/part-six/) पढ़ चुके है।
 
-## What's in this tutorial?
+## इस ट्यूटोरियल में क्या है?
 
-In the previous tutorial, you created a nice index page that queries markdown
-files and produces a list of blog post titles and excerpts. But you don't want to just see excerpts, you want actual pages for your
-markdown files.
+पिछले ट्यूटोरियल में, आपने एक अच्छा इंडेक्स पेज बनाया था, जो मार्कडाउन फाइल्स को query करता है
+और ब्लॉग पोस्ट शीर्षक और अंशों की एक सूची तैयार करता है। लेकिन आप केवल अंश नहीं देखना चाहते हैं,आप मार्कडाउन फाइलें के लिए वास्तविक पेजेज चाहते हैं।
 
 You could continue to create pages by placing React components in `src/pages`. However, you'll
```

Comment:
You missed this?
This also should be translated, Line 14 and 15

Reply:
add a small comment on this

Reply:
This is done. right?